<launch>
	<arg name="tf_prefix" default="" />
    <arg name="gui" default="false" />
    
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
        <param name="use_gui" value="$(arg gui)" />
        <rosparam param="/source_list">[/joint_states]</rosparam>
    </node>
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
        <param name="tf_prefix" value="$(arg tf_prefix)" />
        <rosparam param="/source_list">[/robot_state]</rosparam>
    </node>
</launch>